core/src/main/java/com/arcrobotics/ftclib/controller/PIDFController.java

```diff
@@ -110,6 +110,8 @@ public double getSetPoint() {
      */
     public void setSetPoint(double sp) {
```

Comment:
Should we add a set point for velocity in this as well? And, relating to that, maybe also an acceleration setpoint?